pkg/kudoctl/cmd/convert.go

```diff
@@ -0,0 +1,32 @@
+package cmd
+
+import (
+	"fmt"
+
+	"github.com/kudobuilder/kudo/pkg/kudoctl/cmd/convert"
+	"github.com/spf13/cobra"
+
+	"github.com/kudobuilder/kudo/pkg/kudoctl/util/vars"
+)
+
+// newConvertCmd creates a new command that shows the plans available for an instance
+func newConvertCmd() *cobra.Command {
+	newCmd := &cobra.Command{
+		Use:   "convert",
+		Short: "Converts helm chart into KUDO repo structure",
+		Long:  `Converts the provided helm repo into KUDO structure by modifying `,
```

Comment:
is this line somehow truncated? 🤔 it feels like it ends in the middle of sentence